### PR TITLE
Grid feature

### PR DESCRIPTION
**Pull request by mariyasada**
changing the link of Iqra UI. do review it

### CODE REVIEW

**Comment by Bhakti199** [comment]
everything looks fine to me here.

File: stylesheets/card/card.css [hunk 1]
```diff
@@ -64,9 +64,9 @@
 }
 button
 {
-    width:95%;
+    min-width:100%;
     display: inline-block;
-    margin: 0.3rem 0.7rem;    
+    margin: 0.5rem auto;
```
Comment (Bhakti199) [inline stylesheets/card/card.css:69]: everything looks fine to me.

**Review by Bhakti199 — APPROVED** [review]
UI looks really cool.

**Review by ShreyashMahajan — APPROVED** [review]
looks good to merge

**Pull request merged**